doc/user/clusterwide.md

```diff
@@ -0,0 +1,12 @@
+# Manage clusters in all namespaces
+
+Default etcd operator behavior is to only manage etcd clusters created in the same namespace.
+It is possible to deploy an etcd operator with special option to manage clusterwide etcd clusters.
+
+## Install etcd operator
+
+etcd operator have to run with `-cluster-wide` arg option.
+
+More information in [install guide](doc/user/install_guide.md).
+
+## [Cluster resource example](example/example-etcd-cluster-clusterwide.yaml)
```

Comment:
Probably not use a header?
Add a sentence:

```
See the example in ...
```

Reply:
@guilhem fix this?

Reply:
@fanminshi hehe was in another batch ;)